core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -1107,6 +1107,7 @@ class TimeSeriesShard(val ref: DatasetRef,
     */
   private def addPartitionForIngestion(recordBase: Any, recordOff: Long, schema: Schema, group: Int) = {
     assertThreadName(IngestSchedName)
+    logger.debug(s"Adding part key XXX: ${schema.ingestionSchema.debugString(recordBase, recordOff)}")
```

Comment:
Maybe a better search string than XXX?

Reply:
I suppose we can leave out the XXX, or change it to "details"  :)